pom.xml

```diff
@@ -18,6 +18,7 @@
         <module>calculation-service</module>
         <module>contracts</module>
         <module>aggregate-coverage-report</module>
+        <module>contracts/email-contracts</module>
```

Comment:
This shouldn't be here

Email-contracts should be a sub-module of the 'contracts' module